src/opm/material/components/CO2.cpp

```diff
@@ -23,6 +23,9 @@
 
 #include <config.h>
 #include <opm/material/components/CO2.hpp>
+#if HAVE_QUAD
+#include <opm/material/common/quad.hpp>
+#endif
```

Comment:
The `quad.hpp` file [`#include<>`s](https://github.com/OPM/opm-common/blob/ff2e8c833300ad1217380acd816c753cfeece04c/opm/material/common/quad.hpp#L341) `classname.hh` from Dune.  Therefore, every file that includes `quad.hpp` will need to link to Dune-common to get the appropriate include directories if Dune is installed outside of system directories.

I can obviously set `CMAKE_DISABLE_FIND_QuadMath` in my build configuration, but maybe there's a better way.

```
FAILED: CMakeFiles/opmcommon.dir/src/opm/material/components/CO2.cpp.o 
/usr/bin/g++  -DBOOST_ALL_NO_LIB -DBOOST_SYSTEM_DYN_LINK -DFMT_HEADER_ONLY -DHAVE_CONFIG_H=1 -DHAVE_QUAD=1 -D_GLIBCXX_USE_FLOAT128 -I. -I/home/bska/work/opm/src/opm.kdev/opm-common/external/cjson -Iinclude -I/home/bska/work/opm/src/opm.kdev/opm-common -isystem /home/bska/work/opm/src/opm.kdev/opm-common/external/fmtlib/include -pipe -pedantic -Wall -Wextra -Wcast-align -Wcast-qual -Wchar-subscripts -Wformat-nonliteral -Wmissing-declarations -Wpointer-arith -Wredundant-decls -Wundef -Wwrite-strings -fopenmp -pthread -ggdb3 -O0 -DDEBUG -fPIC   -fopenmp -std=c++17 -fext-numeric-literals -MD -MT CMakeFiles/opmcommon.dir/src/opm/material/components/CO2.cpp.o -MF CMakeFiles/opmcommon.dir/src/opm/material/components/CO2.cpp.o.d -o CMakeFiles/opmcommon.dir/src/opm/material/components/CO2.cpp.o -c /home/bska/work/opm/src/opm.kdev/opm-common/src/opm/material/components/CO2.cpp
In file included from /home/bska/work/opm/src/opm.kdev/opm-common/src/opm/material/components/CO2.cpp:27:
/home/bska/work/opm/src/opm.kdev/opm-common/opm/material/common/quad.hpp:56:5: warning: non-standard suffix on floating constant [-Wpedantic]
   56 |     { return FLT128_MIN; }
      |     ^
/home/bska/work/opm/src/opm.kdev/opm-common/opm/material/common/quad.hpp:58:5: warning: non-standard suffix on floating constant [-Wpedantic]
   58 |     { return FLT128_MAX; }
      |     ^
/home/bska/work/opm/src/opm.kdev/opm-common/opm/material/common/quad.hpp:69:5: warning: non-standard suffix on floating constant [-Wpedantic]
   69 |     { return FLT128_EPSILON; }
      |     ^
/home/bska/work/opm/src/opm.kdev/opm-common/opm/material/common/quad.hpp:90:5: warning: non-standard suffix on floating constant [-Wpedantic]
   90 |     { return FLT128_DENORM_MIN; }
      |     ^
/home/bska/work/opm/src/opm.kdev/opm-common/opm/material/common/quad.hpp:341:10: fatal error: dune/common/classname.hh: No such file or directory
  341 | #include <dune/common/classname.hh>
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
```

Reply:
try putting that classname stuff in quad.hpp inside #if HAVE_DUNE_COMMON.